src/test/java/apicurito/tests/steps/verification/CommonVerifications.java

```diff
@@ -0,0 +1,104 @@
+        private static By QUERY_PARAM_SECTION = By.cssSelector("query-params-section");
+        private static By HEADER_PARAM_SECTION = By.cssSelector("header-params-section");
+        private static By REQUEST_BODY_SECTION = By.cssSelector("requestbody-section");
+        private static By DESCRIPTION = By.className("description");
+        private static By PARAMETERS_TYPE = By.className("param-type");
+    }
+
+    /*
+        param could be header|query|RFD
+        page could be path|operations
+     */
+    @Then("check that exist {string} parameters or RDF on {string} page")
+    public void checkThatExistParametersOnPage(String param, String page, DataTable table) {
+        SelenideElement pageElement = page.equals("operations") ? OperationUtils.getOperationRoot() : PathUtils.getPathPageRoot();
+        String rowType = null;
+        By section = null;
+
+        switch (param) {
+            case "query":
+                rowType = "query-param-row";
+                section = Elements.QUERY_PARAM_SECTION;
+                break;
+            case "header":
+                rowType = "header-param-row";
+                section = Elements.HEADER_PARAM_SECTION;
+                break;
+            case "RFD":
+                rowType = "formdata-param-row";
+                section = Elements.REQUEST_BODY_SECTION;
+                break;
+        }
+
+        CommonUtils.openCollapsedSection(pageElement, section);
+
+        for (List<String> dataRow : table.cells()) {
+            checkRow(dataRow, pageElement, section, rowType, param + " parameter");
+        }
+    }
+
+    private void checkRow(List<String> dataRow, SelenideElement pageElement, By section, String rowType, String message) {
+        ElementsCollection queryRows = pageElement.$(section).$$(rowType);
+
+        if (queryRows.size() > 0) {
+            ElementsCollection names = pageElement.$(section).$$(By.className("name")).filter((text(dataRow.get(0))));
+
+            if (names.size() == 0) {
+                fail("%s with name %s is not created!", message, dataRow.get(0));
+            } else {
+                SelenideElement row = names.first().parent().parent();
+
+                if (!dataRow.get(1).isEmpty()) {
+                    String description = row.$(Elements.DESCRIPTION).getText();
+                    assertThat(description).as("%s description should be %s but is %s", message, dataRow.get(1), description).isEqualTo(dataRow.get(1));
+                }
+
+                row.$(By.className("summary")).click();
+
+                if (!dataRow.get(2).isEmpty()) {
+                    String isRequired = row.$(By.className("param-required")).$("drop-down").getText();
+                    assertThat(isRequired)
+                            .as("%s should be %s but is %s", message, dataRow.get(2), isRequired)
+                            .isEqualTo(dataRow.get(2));
+                }
+
+                if (!dataRow.get(3).isEmpty()) {
```

Comment:
This and the following two steps seem to be quite repetitive, could it be optimized?

Reply:
It will be part of the next PR as we discussed.